src/data-and-backend/serialization/json.md

```diff
@@ -61,6 +61,12 @@ manual serialization might be the way you want to start.
 For an example of manual encoding, see
 [Serializing JSON manually using dart:convert][].
 
+{{site.alert.tip}}
+  For hands-on practice manually deserializing JSON and
```

Comment:
**suggestion**: Remove "manually". With "hands-on", it's redundant.

Reply:
Good point, thanks!

```suggestion
  For hands-on practice deserializing JSON and
```